src/components/CreateCredential.js

```diff
@@ -17,53 +20,131 @@ const CreateCredential = () => {
 
+      <GetCredential.Return title="Previously created projects" paragraph="Select a project and access your existing credentials for Firefly - Firefly and Creative Cloud Automation." nextStepsLabel="Next steps" nextStepsHref="/credentials/nextsteps" className="card_developer_console">
+
+        <GetCredential.Return.Side>
+          <GetCredential.Return.Side.Custom>
+            <div style={{ display: "flex", gap: "30px", flexDirection: "column", width: "100%" }}>
+              <h3 className='spectrum-Heading spectrum-Heading--sizeM'>Welcome back</h3>
+              <p className="spectrum-Body spectrum-Body--sizeM">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation.</p>
+            </div>
+          </GetCredential.Return.Side.Custom>
+          <GetCredential.Return.Side.NewCredential heading="Need another credential?" buttonLabel="Create new credential" />
+        </GetCredential.Return.Side>
+
+        <GetCredential.Return.CredentialDetails heading="Credential details">
+          <GetCredential.Return.CredentialDetails.APIKey heading="API Key" />
+          <GetCredential.Return.CredentialDetails.AllowedOrigins heading="Allowed domains" />
+          <GetCredential.Return.CredentialDetails.OrganizationName heading="Organization" />
+        </GetCredential.Return.CredentialDetails>
+
+        <GetCredential.Return.ProjectsDropdown label="Projects" subHeading="Only your projects that contain credentials are shown" />
+
+        <GetCredential.Return.ManageDeveloperConsole label="Manage all your projects and credentials on Adobe Developer Console" direction='/console/projects' />
+
+        <GetCredential.Return.DevConsoleLink heading="Developer Console Project" />
+
+        <GetCredential.Return.Products label="Included products and services">
+          <GetCredential.Return.Product label="Adobe Express Embed SDK" icon={creativeCloud} />
+        </GetCredential.Return.Products>
+
+      </GetCredential.Return>
+
+      <GetCredential.RequestAccess
+        title="Get credentials"
+        paragraph="Create unique credentials that you will use to call multiple APIs from your application."
+      >
+        <GetCredential.RequestAccess.EdgeCase>
+          <GetCredential.RequestAccess.EdgeCase.NoProduct title="Your organization does not have access to Adobe Express Embed SDK" buttonLabel="Contact us to learn more" buttonLink="#someLink" />
+          <GetCredential.RequestAccess.EdgeCase.NotMember title="Access to Adobe Express Embed SDK APIs is not available at this time." buttonLabel="Learn more about Adobe Express Embed SDK" buttonLink="#someLink" />
+          <GetCredential.RequestAccess.EdgeCase.NotSignUp title="Adobe Express Embed SDK APIs is available as part of the beta program. Sign up for the program or log in to an account that has access." buttonLabel="Sign up for the beta" buttonLink="#someLink" />
```

Comment:
This case is not applicable for express embed sdk. Can we remove this?

Reply:
Yeah , Sure